frontend/src/pages/question/panels/EdgePanel.jsx

```diff
@@ -34,9 +34,9 @@ export default function EdgePanel(props) {
     }
     return Object.entries(biolink.slots).map(
       ([identifier, predicate]) => ({
-        name: biolinkUtils.snakeCase(identifier),
-        domain: biolinkUtils.snakeCase(predicate.domain),
-        range: biolinkUtils.snakeCase(predicate.range),
+        label: strings.toSnakeCase(identifier),
```

Comment:
Predicate drop-down does not seem to be working:


https://user-images.githubusercontent.com/1634866/105096486-226a5700-5a75-11eb-81e6-4d0c31bfc8d6.mov

Reply:
There are some other bugs with predicates as well. Since this is already a large PR and technically specifically for node types, I am going to create another issue for predicates and open its own PR.